README.md

```diff
@@ -1,78 +1,108 @@
 # annlink
 Link between eralgn applications and deep learning frameworks.
 
-# Installation
+## Installation
 
 ```
-git clone https://github.com/K2InformaticsGmbH/annlink.git
+1. install the Apche Thrift compiler (see https://thrift.apache.org/ Build and Install the Apache Thrift compiler)
+2. install a Python 3 environment including the packages numpy, pandas and tensorflow
+3. git clone https://github.com/K2InformaticsGmbH/annlink
+4. cd annlink
+5. thrift_compile.bat (generates the sources from the Thrift files)  
+6. rebar3 compile
+7. cd ..
+8. git clone https://github.com/apache/thrift
+9. cd thrift/lib/py
+10. python setup.py install
+11. copy the whole directory thrift\lib\py\build\lib.win-amd64-3.6\thrift to directory annlink\src_py 
```

Comment:
This is not necessary I think as the setup step should install the package into your environment.